src/adapter/src/coord/consistency.rs

```diff
@@ -73,10 +82,45 @@ impl Coordinator {
             Err(read_capabilities_inconsistencies)
         }
     }
+
+    /// # Invariants
+    ///
+    /// * All [`GlobalId`]s in the `active_webhooks` map should reference known webhook sources.
+    ///
+    fn check_active_webhooks(&self) -> Result<(), Vec<ActiveWebhookInconsistency>> {
+        let mut inconsistencies = vec![];
+        for (id, _) in &self.active_webhooks {
+            let is_webhook = self
+                .catalog()
+                .try_get_entry(id)
+                .map(|entry| entry.item())
+                .and_then(|item| {
+                    let CatalogItem::Source(Source { data_source, .. }) = &item else {
+                        return None;
+                    };
+                    Some(matches!(data_source, DataSourceDesc::Webhook { .. }))
+                })
+                .unwrap_or(false);
+            if !is_webhook {
+                inconsistencies.push(ActiveWebhookInconsistency::NonExistentWebhook(*id));
```

Comment:
Uncovered

Reply:
same as above, this is a consistency check